src/common/services/api/profile.service.js

```diff
@@ -81,6 +81,7 @@ class Profile {
       .map((donorDetails) => {
         donorDetails.mailingAddress = formatAddressForTemplate(donorDetails['mailing-address'].address)
         delete donorDetails['mailing-address']
+        donorDetails['registration-state'] = 'NEW'
```

Comment:
I'm not sure I understand why this was added. The registration state should come from the API typically. Anyone coming to the profile page should have a registration state (post login steps).

Reply:
I agree. I'm unsure why I added it so I'll remove it and test to ensure everything is working as expected.